### DISCUSSION TITLE
为什么红莲之狂潮有些任务的一部分没有汉化？简体中文

### DISCUSSION THREAD

**Discussion by BangCockk** [GpointChen/FFXIVChnTextPatch-GP#368] [Q&A]
有没有办法解决这个问题？

**Comment by GpointChen** [comment]
有截圖嗎？方便我確認。

**Reply by BangCockk** [reply]
我下回出现这种问题的时候给你发吧，很随机的这个

**Reply by BangCockk** [reply]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/73ffe249-2d7f-40b2-afa5-f8e2a69935d2)
这个是贤者的任务
贤者的进化

**Reply by BangCockk** [reply]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/d6bd8519-4502-4162-8700-38e0289737f8)
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/52d9ae84-1e73-42c6-a833-f29830caf670)

**Reply by BangCockk** [reply]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134466621/2beb6e6f-575a-4736-a568-b4100d6819bd)

**Comment by Liteya** [comment]
因為有部分資料更動及追加了 .....

**Comment by BangCockk** [comment]
还有任务 红莲之狂潮 没有汉化，都是英文
